package.json

```diff
@@ -71,11 +72,6 @@
       "@oclif/plugin-update",
       "@oclif/plugin-warn-if-update-available"
     ],
-    "update": {
```

Comment:
why are you removing all these?

Reply:
I used 'yarn install' to add dynamodb package, not sure why it changed some other things